Tools/Benchmarks/STREAM/sample.json

```diff
@@ -0,0 +1,22 @@
+{
+    "guid": "39AA03BC-09ED-4DDF-A115-FD31F6F46450",
+    "name": "STREAM",
+    "categories": ["Toolkit/oneAPI Tools/Benchmarks"],
+    "description": "The STREAM is a program that measures memory transfer rates in MB/s for simple computational kernels coded in C",
+    "toolchain": [ "dpcpp" ],
+    "targetDevice": [ "CPU", "GPU" ],
+    "languages": [ { "cpp": {} } ],
+    "os": [ "linux" ],
+    "builder": [ "ide", "cmake" ],
```

Comment:
Since this sample is intended only for Linux, the "ide" reference in the "builder" attribute will be ignored. It is only used by Visual Studio on Windows platforms (see https://github.com/oneapi-src/oneAPI-samples/wiki/sample-browser-behavior#builder for details).

It's not required that this be fixed, but be aware that if you do update this sample to be supported on Windows it means that VS proj and solution files need to also be provided with the sample.